syntax/std_eval.go

```diff
@@ -13,9 +17,13 @@ func stdEval() rel.Attr {
 }
 
 func evalExpr(v rel.Value) rel.Value {
-	evaluated, err := EvaluateExpr(".", v.(rel.String).String())
-	if err != nil {
-		panic(err)
+	switch val := v.(type) {
+	case rel.String, rel.Bytes:
+		evaluated, err := EvaluateExpr(".", val.String())
+		if err != nil {
+			panic(err)
+		}
+		return evaluated
 	}
-	return evaluated
+	panic(fmt.Sprintf("eval.value only takes byte array or string: %T", v))
```

Comment:
Clearer error message?
```suggestion
	panic(fmt.Sprintf("eval.value only takes byte array or string, received %T", v))
```